src/components/sidebar/linkView/LinkView.tsx

```diff
@@ -68,26 +117,32 @@ class LinkView extends React.Component<ILinkViewProps, ILinkViewState> {
                     <div className={s.flexInnerRow}>
                         <InputContainer
                             label='REITTITUNNUS'
-                            placeholder='1016'
+                            value={this.state.routePathLink!.routeId}
                         />
                         <InputContainer
                             label='SUUNTA'
-                            placeholder='Suunta 1'
+                            value={`Suunta ${routePath ? routePath.direction : '?'}`}
                         />
                     </div>
                     <div className={s.flexInnerRow}>
                         <InputContainer
                             label='VOIM. AST'
-                            placeholder='01.09.2017'
+                            value={
+                                routePath ? moment(
```

Comment:
Same comment as for routeid. This is part of a composite foreign key

Reply:
disabled